docs/embedding.md

```diff
@@ -27,7 +27,7 @@ Before we dive into the code, let's take a look at some of the key concepts that
 
 ### Stubs
 
-A Stub is the most basic concept in Modal. We can declare it using just 2 lines of code.
+A Stub is the most basic concept in Modal — it's a description of how to create a Modal application. We can declare it using just 2 lines of code.
```

Comment:
This is a nice way to put it and is consistent with the modal docs too for a stub docs - https://modal.com/docs/reference/modal.Stub.